stable/artifactory-ha/templates/nginx-deployment.yaml

```diff
@@ -125,7 +125,8 @@ spec:
         {{- end }}
         {{- if .Values.nginx.customArtifactoryConfigMap }}
         - name: artifactory-nginx-config
-          mountPath: "/tmp/conf.d"
+          mountPath: "/etc/nginx/conf.d/artifactory-ha.conf"
+          subPath: "artifactory-ha.conf"
```

Comment:
Shouldn’t this be configurable? Or at least documented? I mean, the fact that you have to have a key called artifactory-ha.conf here and artifactory.conf in the artifactory chart

Reply:
@danielezer  https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/ReverseProxyConfiguration.md#steps-to-use-static-configuration-for-reverse-proxy-in-nginx

Reply:
I see that but I think it’s worth mentioning that the data key has to be called by that specific name. I’m not I would understand that based in the current documentation. But that’s just me 😅

Reply:
@danielezer I do agree. Do you want me to make it configurable or just add note in the readme?

Reply:
Mmm whatever you think 🙂
I think both are good

Reply:
@jainishshah17 what about this change? what do you think?

Reply:
@danielezer Done. PTAL.